include/common.sh

```diff
@@ -790,7 +792,7 @@ printError()
   printColors "${NC}${RED}${BOLD}***ERROR: ${NC}${RED}${1}${NC}" >&2
   [ -n "${xtrc}" ] && set -x
   mutexFree "zopen" # prevent lock from lingering around after an error
-  cleanupFunction
+cleanupFunction
```

Comment:
Is this change intended?

Reply:
No, just me not spotting the indentation change as "show whitespace differences" wasn't enabled when I reinstated the cleanup call (as was testing without it)